protocol/avdl/chat1/common.avdl

```diff
@@ -420,6 +420,9 @@
     // ephemeral lifetime calculations
     @mpackkey("rt") @jsonkey("rt")
     gregor1.Time rtime;
+
+    @mpackkey("pm") @jsonkey("pm")
```

Comment:
Why not boot this guy too? Juts make `HasPairwiseMacs` do what we do to set the bool in boxer.

Reply:
i think it's required here since we need to copy from the `MessageClientHeader` to `MessageClientHeaderVerified` in some form. I'm not sure what bool you're referring to but calling `ShouldPairwiseMAC` https://github.com/keybase/client/blob/e8e11b3639573f66cc45b1730fb5f3f17a73fc69/go/chat/boxer.go#L1336 would be expensive